app/src/containers/ViewProfile/index.js

```diff
@@ -15,11 +15,10 @@ import { Redirect } from 'react-router';
 import { History } from 'react-router';
 
 //Components
-import {AlertView} from '../../components/AlertView'
+import {AlertView, Alert} from '../../components/AlertView'
```

Comment:
Consider changing the naming here to distinguish between the corner snack bar message and the center long bar message